components/search/SearchForm.vue

```diff
@@ -230,14 +231,14 @@
 
         this.gettingSuggestions = true;
 
-        // Query in the user's language, and English, removing duplicates
-        const languageParam = Array.from(new Set([this.$i18n.locale, 'en'])).join(',');
-
         const suggestions = await getEntitySuggestions(query, {
-          language: languageParam
+          language: this.$i18n.locale
         }, {
           recordValidation: this.enableSuggestionValidation
-        });
+        })
+          .catch((error) => {
+            throw apiError(error);
```

Comment:
When errors are encountered from the entity suggestion endpoint, we do not want to pester the user about them, replacing the page content with an error message, but instead should just show no suggestions for their query. This should not re-throw the error, but should set suggestions to an empty array.

Reply:
OK understood, I have instead replaced this with 
`this.isAutoSuggestActive;`
after `getEntitySuggestions`

So autosuggest is hidden if no suggestions are returned